python/rikai/spark/sql/codegen/mlflow_logger.py

```diff
@@ -75,23 +75,28 @@ def log_model(
         kwargs: dict
             Passed to `mlflow.<flavor>.log_model`
 
+
         Examples
         --------
-        # import mlflow.pytorch  <= vanilla mlflow usage
-        import rikai.mlflow
 
-        # Log PyTorch model
-        with mlflow.start_run() as run:
+        .. code-block:: python
+
+            # import mlflow.pytorch  <= vanilla mlflow usage
```

Comment:
i forgot why this is there. should we just remove this line?

Reply:
ok, was just fixed the indent. lemme remove this line then.